petab/sbml.py

```diff
@@ -217,7 +221,19 @@ def add_global_parameter(sbml_model: libsbml.Model,
                          constant: bool = False,
                          units: str = 'dimensionless',
                          value: float = 0.0) -> libsbml.Parameter:
-    """Add new global parameter to SBML model"""
+    """Add new global parameter to SBML model
+
+    Arguments:
```

Comment:
```suggestion
    Parameters:
```

Reply:
Currently "Arguments" is used in most places. Both do the job equally well I think. Having some docstring-guideline somewhere wouldn't hurt, though.

Reply:
yeah I was not sure if this works properly with sphinx. seems to be fine though, automatically translated to a "Parameters" field it seems ...